Makefile

```diff
@@ -1,18 +0,0 @@
-SRC = $(wildcard src/*.js)
```

Comment:
You are correct here, but Makefile is my personal preference. I know I can handle it another way, but I use Makefile in all my projects.